test/SemVer.unittest.js

```diff
@@ -130,11 +130,17 @@ describe("SemVer", () => {
 	describe("parseRange", () => {
 		const cases = {
 			"=3": ["3", "v3", "3.x", "3.X", "3.x.x", "3.*", "3.*.*", "^3", "^3.x"],
+			"= 3": ["3", "v3", "3.x", "3.X", "3.x.x", "3.*", "3.*.*", "^3", "^3.x"],
```

Comment:
```suggestion
			"=3": ["3", "v3", "3.x", "3.X", "3.x.x", "3.*", "3.*.*", "^3", "^3.x", "= 3"],
```

The key is the "expected" value and all values are tested to be equal that.

Reply:
Oh, this makes a lot of sense. I've adjusted the tests.